doc/architectural/folder-walkthrough.md

```diff
@@ -75,14 +75,10 @@ into the `doc/html` directory when running `doxygen` from `src`.
 
 ## `regression/` ##
 
-The `regression/` directory contains the test suites.
-They are grouped into directories for each of the tools/modules.
-Each of these contains a directory per test case, containing a C or C++
-file that triggers the bug and a `.desc` file that describes
-the tests, expected output and so on. There is a Perl script,
-`test.pl` that is used to invoke the tests as:
+The `regression/` directory contains the regression test suites. How to run the
+regression tests is documented in \ref compilation-and-development
```

Comment:
I would have chosen slightly different wording: "See \ref compilation-and-development for information on how to run and develop regression tests." But then English isn't my first language, so please take with a grain of salt.

Reply:
Yes, that is more natural.